src/core/inlines.js

```diff
@@ -124,3 +124,31 @@ export function run(conf) {
     txt.parentNode.replaceChild(df, txt);
   }
 }
+
+function isInformative(ref, parentNode) {
+  const informSelectors = ".informative, .note, figure, .example, .issue";
+  const closestInformative = parentNode.closest(informSelectors);
+
+  let informative = false;
+  if (closestInformative) {
+    // check if parent is not normative
+    if (!parentNode.closest(".normative")) {
+      informative = true;
```

Comment:
The two statements inside the ifs could be combined into:

informative = stament1 || statement2; 

Wdyt? In either case, the second if is getting run even if informative is already true.

Reply:
Yes. I was thinking it was cleaner (I do not enjoy multiple if statements. but looks like nested if's are also a no-go)

Reply:
I guess the main thing is not to run the second if statement. It could be short circuited if the first if statement sets `informative` to true.

Reply:
Yes. I missed this part earlier. Fix now